pkg/debug/scripts/collect-info.sh

```diff
@@ -220,6 +220,7 @@ collect_network_info()
                    > "$DIR/network/iptables-mangle"
     iptables -L -v -n --line-numbers -t nat \
                    > "$DIR/network/iptables-nat"
+    eve exec pillar /opt/zededa/bin/conntrack > "$DIR/network/conntrack"
```

Comment:
Do we have this tool in all the EVE versions? Or should we handle the situation when it's not there?